src/odemis/driver/pigcs.py

```diff
@@ -867,11 +869,12 @@ def IsOnTarget(self, axis, check=True):
             if err:
                 raise PIGCSError(err)
             r = lresp[1]
-            ss = r.split("=")
+            ss = r.split(b"=")
             if len(ss) != 2:
                 raise ValueError("Failed to parse answer from %s: '%s'" %
-                                 (com, lresp))
-            return ss[1] == "1"
+                                 ([c.decode('latin1', 'backslashreplace') for c in com],
+                                  lresp.decode('latin1', 'backslashreplace')))
```

Comment:
`lresp` is also a list. => also a list comprehension. Alternatively, you could just use "%r" and be done ;-) We don't need too much beauty in this very unlikely error message.